sql/core/src/test/resources/sql-tests/results/postgreSQL/numeric.sql.out

```diff
@@ -3831,12 +3831,12 @@ struct<>
 -- !query output
 org.apache.spark.sql.AnalysisException
 {
-  "errorClass" : "_LEGACY_ERROR_TEMP_1168",
+  "errorClass" : "NUM_COLUMNS_MISMATCH",
   "messageParameters" : {
-    "insertedColumns" : "5",
-    "staticPartCols" : "0",
-    "tableName" : "`spark_catalog`.`default`.`num_result`",
-    "targetColumns" : "3"
+    "firstNumColumns" : "3",
+    "invalidNumColumns" : "5",
+    "invalidOrdinalNum" : "second",
```

Comment:
@cloud-fan @srielau Are you ok that we are talking about inputs (and their ordinal number), but not a table name.

Reply:
Not really Ok with that. I'm also confused by the "second" portion. That doesn't translate and seems unnecessary compared to "1." and "2.".
Should we have a dedicated error message? Especially INSERT INTO has an optional column list, so there are many ways to "fix" the error. It's not the same as a UNION ALL or an EXCEPT.

Reply:
@panbingkun Any update on this? Can we have this on Apache Spark 3.4 ?